crates/katana/runner/src/logs.rs

```diff
@@ -0,0 +1,92 @@
+use std::fs::File;
+use std::io::{BufRead, BufReader};
+use std::time::Duration;
+
+use chrono::prelude::*;
+use tokio::time::sleep;
+
+use crate::KatanaRunner;
+
+impl KatanaRunner {
```

Comment:
these should all use json logs too

Reply:
Most of the logs just have the same message field, but done